os/guest/setup-raspbian.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+#
+## === Actions ===
+#
+
+# force pi user to change password upon first login
+chage -d 0 pi
```

Comment:
"change" ?

Reply:
[ch]ange[age]: make the age of the password 0, meaning expired. See `man chage`.